book/website/reproducible-research/overview/overview-barriers.md

```diff
@@ -0,0 +1,155 @@
+# Barriers to reproducibility
+
+So far we have [explained what we mean](overview-definitions) by reproducible research and explained some of the [additional benefits](overview-benefits).
+
+But there are many barriers to reproducible research.
+
+You can watch Kirstie Whitaker describe some of them in [her talk about _The Turing Way_](https://youtu.be/wZeoZaIV0VE?t=312) at [csv,conf,v4](https://csvconf.com/2019) in May 2019.
+You can use and re-use her slides under a CC-BY licence via Zenodo (doi: [10.5281/zenodo.2669547](https://doi.org/10.5281/zenodo.2669547)).
+The section describing the slide below starts around 5 minutes into the video.
+
+```{figure} ../../figures/barriers-reproducibility.png
+---
+width: 500px
+name: reproducibility-barriers
+alt: Slide from the presentation showing the different barriers to reproducibility. The text in the center says 'Barriers to reproducible research' and the following barriers are arranged clockwise around the slide - Is not considered for promotion, Held to a higher standard than others, Publication bias towards novel findings, Plead the 5th, Takes time, Support additional users, Requires additional skills.
+---
+A slide outlining some of the barriers to reproducible research.
+```
+
+This chapter outlines some of those barriers, and a couple of suggestions to get around them.
+
+- [Plead the fifth](#plead-the-fifth)
+- [Publication bias towards novel findings](#publication-bias-towards-novel-findings)
+- [Held to higher standards than others](#held-to-higher-standards-than-others)
+- [Not considered for promotion](#not-considered-for-promotion)
+- [Big data and complex computational infrastructure](#big-data-and-complex-computational-infrastructure)
+- [Being reproducible does not mean the answer is right](#being-reproducible-does-not-mean-the-answer-is-right)
+- [Takes time](#Takestime)
+- [Support additional users](#support-additional-users)
+- [Requires additional skills](#requires-additional-skills)
+
+## Plead the fifth
```

Comment:
Would recommend that this is updated with a more explanatory heading for non-US folks (I know there's a definition below but I still think it would be helpful)

Reply:
Gave it a re-write - thank you for the check!